internal/commands/vorpal-engine.go

```diff
@@ -0,0 +1,125 @@
+package commands
+
+import (
+	"path/filepath"
+
+	"github.com/checkmarx/ast-cli/internal/commands/util/printer"
+	errorConstants "github.com/checkmarx/ast-cli/internal/constants/errors"
+	commonParams "github.com/checkmarx/ast-cli/internal/params"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func runScanVorpalCommand() func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		vorpalLatestVersion, _ := cmd.Flags().GetBool(commonParams.VorpalLatestVersion)
+		sourceFlag, _ := cmd.Flags().GetString(commonParams.SourcesFlag)
+
+		scanResult, err := ExecuteVorpalScan(sourceFlag, vorpalLatestVersion)
+		if err != nil {
+			return err
```

Comment:
return err print the err?

Reply:
We print the successful response using the printer because the wrappers need to pick it from the console.
the error is returned because this is the function signature in all of our command and it's printed to the console.